IFC4x3/Properties/p/PositiveDistanceAlongDirection_1A$$mWrV50AwsrskfY48Rv/Documentation.md

```diff
@@ -2,3 +2,5 @@ Specifies the direction along the linear element being measured considered as po
 for negative.
 
 > NOTE&nbsp;Definition according to ISO 19148:2021.
+
+> NOTE&nbsp;The positive direction is defined by the increasing length of the _IfcLinearPositioningElement_ (e.g. _IfcAlignment_) being referenced.
```

Comment:
This will not work.

Reply:
Can this work?:
- Remove these (offsetLateral/-Vertical) from Pset_Stationing
- Introduce a Pset_PositionExpression (applicable to IfcReferent/POSITION)
  - Station
  - OffsetLateralDistance (the direction is dependent on the stationing - Pset_Stationing)
  - OffsetVerticalDistance (the direction is dependent on the stationing - Pset_Stationing)
 Other suggestions (besides skipping it completely)?

Reply:
Station is probably an IfcLabel?

For both offsets I still see the same issue. What is lateral and what vertical? Would those be dependent on the earth surface / IfcGeometricRepresentationContext.WorldCoordinateSystem and lateral would be to the left / right of the positive measurements? But what are positive measurements since stations are merely text? Would and alphanumerical comparison of strings determine what is positive?

I see this as a very complex implementation. That is why we have axis2placement (and its types).

I am quite skeptical in dealing with this in psets.